backend/src/utils/permissions.ts

```diff
@@ -1,4 +1,4 @@
-import { adminCollection } from './firebase';
+import { adminCollection } from '../firebase';
 
 export default class PermissionsManager {
```

Comment:
Maybe this file should just be renamed to `permissionsManager.ts`

Reply:
done!